README.md

```diff
@@ -53,26 +53,30 @@ any new files in the repository.
 Make your own customizations
 ----------------------------
 
-Put your customizations in dotfiles appended with `.local`:
+Create a directory for your personal customizations: 
 
-* `~/.aliases.local`
-* `~/.git_template.local/*`
-* `~/.gitconfig.local`
-* `~/.gvimrc.local`
-* `~/.psqlrc.local` (we supply a blank `.psqlrc.local` to prevent `psql` from
+    mkdir ~/dotfiles-local
```

Comment:
I don't personally use `~/dotfiles-local` as my directory. I use:

```
~/src/github.com/croaky/dotfiles
~/src/github.com/thoughtbot/dotfiles
```

https://robots.thoughtbot.com/manage-team-and-personal-dotfiles-together-with-rcm